libnetwork/iptables/iptables.go

```diff
@@ -173,6 +173,24 @@ func (iptable IPTable) LoopbackByVersion() string {
 	return "127.0.0.0/8"
 }
 
+// SetupInternalICC is used to add or remove the internal interface from the firewalld zone ( ICC )
+// https://github.com/moby/libnetwork/issues/2647
```

Comment:
I suggest adding a comment to clarify the purpose of this method to avoid confusion in the future